index.html

```diff
@@ -17,6 +17,7 @@
 				rootSelector: '#app',
 				licenseUrl: '',
 				licenseName: '',
+				wikibaseLexemeTermLanguages: [ 'en' ],
```

Comment:
Could we have a lot more here? Maybe like 10 or so.

Then https://ed85c311d1a49ef3e851d47e436a740779b1--new-lexeme-special-page.netlify.app/ is more useful

Reply:
you are right. will add a few